pages/design/rule-design.md

```diff
@@ -63,6 +63,21 @@ The description is a short synopsis of what the rule does. This should be declar
 
 List all accessibility requirements that are not satisfied if one or more outcomes of a rule fails. For example, this could be one or more WCAG success criteria, or a WAI-ARIA rule. Each requirement should indicate the associated conformance level, and map the implications of each outcome, such as "satisfied", "not satisfied", or "further testing needed".
 
+Accessibility requirements should be listed as secondary if the requirement is related to the rule. There are three scenarios:
+
+1. The requirement is less strict than the rule. If this occurs, and some but not all failed examples of the rule pass the accessibility requirement, the following test should be used:
+
+   > This success criterion is **less strict** than this rule. This is because {reason}. Some of the failed examples satisfy this success criterion.
```

Comment:
```suggestion
   > This success criterion is **less strict** than this rule. This is because {reason}. Some of the failed examples may satisfy this success criterion.
```

Reply:
Discussed, not using "may".